### PR TITLE
TW-1299: Support create chat when click on pencil in chat blank

### PR DESCRIPTION
**Pull request by nqhhdev**
### Ticket 

- #1299 

### Resolved

https://github.com/linagora/twake-on-matrix/assets/99852347/355777f2-ae1f-42ea-a0b1-0bca19a3f312

### CODE REVIEW

File: lib/pages/chat_blank/chat_blank.dart [hunk 1]
```diff
@@ -95,18 +99,64 @@ class _ChatBlankRichText extends StatelessWidget {
...
+    if (!goRouteAvailableInFirstColumn) {
+      return FirstColumnInnerRoutes
+          .innerNavigatorNotOneColumnKey.currentState!.context;
+    } else {
+      return FirstColumnInnerRoutes
+          .innerNavigatorOneColumnKey.currentState!.context;
+    }
+  }
+
+  void goToNewGroupChat() {
```
Comment (imGok) [inline lib/pages/chat_blank/chat_blank.dart:145]: you can't `GoToGroupChatMixin` file or adapt it ?

Comment (nqhhdev) [reply lib/pages/chat_blank/chat_blank.dart:145]: I need to manage them separately at this screen, because it's special.

Comment (hoangdat) [reply lib/pages/chat_blank/chat_blank.dart:145]: what is the special here?

Comment (nqhhdev) [reply lib/pages/chat_blank/chat_blank.dart]: Updated use `GoToGroupChatMixin `

**Review by hoangdat — APPROVED** [review]

**Review by sherlockvn — APPROVED** [review]

**Pull request merged**